### PR TITLE
metrics improvements

### PR DESCRIPTION
**Pull request by agardou**
links https://github.com/Opla/backend/issues/27

### CODE REVIEW

File: src/models/users.js [hunk 1]
```diff
@@ -30,7 +34,7 @@ export default class extends AbstractModel {
         email: user.email,
         creation_date: user.creation_date,
         userId: user.id,
-        avatar: user.avatar || "default",
+        avatar: user.avatar || "default"
```
Comment (mikbry) [inline src/models/users.js]: Missing trailing comma . See:
https://eslint.org/docs/rules/comma-dangle

Comment (willdurand) [reply src/models/users.js]: Yup, I think running `yarn lint --fix` will fix all of these things.

Comment (agardou) [reply src/models/users.js]: i don't know why but webstorm remove trailing comma when "reformat with prettier" ??? i will take care of this

Comment (agardou) [reply src/models/users.js]: i have other errors on master with yarn lint : 
on src/index.js
8:20  error  Unable to resolve path to module './config'  import/no-unresolved
8:20  error  Missing file extension for "./config"        import/extensions

Comment (agardou) [reply src/models/users.js]: and comma missing fixed

**Pull request merged**